configs/test/project.yaml

```diff
@@ -70,6 +70,9 @@ env:
   # the testcases against the target application.
   FUZZ_LOGS_BUCKET: test-fuzz-logs-bucket
 
+  # Default bucket to store mutator plugins.
+  MUTATOR_PLUGINS_BUCKET: test-mutator-plugins-bucket
```

Comment:
I suppose this CL is the last chance to decide if we prefer `mutator-plugins` or `mutator-plugin` as the standard bucket name. Wer're not really consistent about this, ie: corpus (singular) vs logs (plural).